### ISSUE TITLE
cross domain images support

### ISSUE THREAD

**Issue by jarben** [ecomfe/echarts-gl#36]
Just started playing with echart-gl - amazing!!!

One thing I've noticed is that you can't load images from other domains, returns: 
Failed to execute 'texImage2D' on 'WebGLRenderingContext': The cross-origin image at xxxx may not be loaded.

Would it be possible to add img.crossOrigin = "Anonymous"; when loading images?

**Comment by pissang** [comment]
Yes it's the limitation of WebGL. Really bothering. I suggest you using a proxy server to load the image.

And thanks!! Any suggestion will be appreciate.

**Comment by jarben** [comment]
Oh, didn't realize img.crossOrigin doesn't work in WebGL, that's pity! 

Just want to say one more time that this echarts-gl is so amazing! I wonder what's the roadmap around it?

**Comment by pissang** [comment]
Sorry for the miss of roadmap, it's still on my computer ;)

There are still many things to do before 1.0.0 is released. Here are some big parts.

+ Transition and animation

+ Integrate with `brush` component.

+ Better view control, orthographic view.

+ Events.

+ Pose mode and screenshots.

And other small improments.

And english doc and examples.

**Comment by pissang** [comment]
About English doc. I'm wondering if you are interested in helping us translate the doc into English ?

**Comment by jarben** [comment]
Definitely happy to help with translation @pissang , that's this repo isn't it? https://github.com/ecomfe/echarts-doc/tree/gh-pages/en/option-gl? 

Regarding roadmap, sounds great! If I can emphasise something it's the Events as it makes it more extensible. Also, handling tooltips and providing necessary data for formatters is a big one! 

On a side note, any thoughts around exposing some qtek API in the similar way how graphic:[] are done? Would be awesome to play with some custom objects - 3d rectangles, texts! But's probably far beyond 1.0:)

**Comment by pissang** [comment]
Glad to hear that!  Yeah that's where English doc is put. And the Chinese doc is https://github.com/ecomfe/echarts-doc/tree/gh-pages/cn/option-gl . Basically we extend markdown with JavaScript template.

And about 3D graphics API. It's in the plan. We will provide a component `graphicGL`, through which, users can add a cube, sphere or even load 3D models conveniently. It will be after 1.0, but not that far ;)

**Comment by jarben** [comment]
That's sounds great, looking forward to play with that!

Regarding translation, will try to allocate some time for this early in May. Also I wonder whether it's now good time for the translation when the Chinese original is still evolving? Or is the API considered almost finished?

**Comment by pissang** [comment]
That will be great! API of the finished components and features will not change much.
